tuna/utils/utility.py

```diff
@@ -118,6 +119,7 @@ def get_env_vars():
   return env_vars
 
 
+# pylint: disable=dangerous-default-value ; @ chris, might want to reconsider though
```

Comment:
If we're never using the parameter to update a supplied dict, we should just remove it and always use the return value.